### PR TITLE
Update CI release scripts to publish for 2.13.0-M5

### PR DESCRIPTION
**Pull request by olafurpg**
Drops 2.13.0-M3 from the build itself, if there is demand for it I can
cut a manual release from my local machine.

### CODE REVIEW

**Comment by olafurpg** [comment]
After merging this we can publish v0.6.6 for 2.13.0-M5, fixing #183

File: bin/testRelease.sh [hunk 1]
```diff
@@ -5,14 +5,14 @@ VERSION=$1
 coursier fetch com.lihaoyi:utest_2.10:$VERSION
 coursier fetch com.lihaoyi:utest_2.11:$VERSION
 coursier fetch com.lihaoyi:utest_2.12:$VERSION
-coursier fetch com.lihaoyi:utest_2.13.0-M3:$VERSION
 coursier fetch com.lihaoyi:utest_2.13.0-M5:$VERSION
+coursier fetch com.lihaoyi:utest_2.13.0-M7:$VERSION
```
Comment (alexarchambault) [inline bin/testRelease.sh]: `s/M7/M5/`?

Comment (alexarchambault) [reply bin/testRelease.sh]: Or just don't add this line

Comment (olafurpg) [reply bin/testRelease.sh]: Fixed. This script is only used to sanity check a release went through.

**Pull request merged**